cmd/nex/node.go

```diff
@@ -152,7 +158,73 @@ func (l List) Run(ctx context.Context, globals *Globals) error {
 	if globals.Check {
 		return printTable("Node List Configuration", append(globals.Table(), l.Table()...)...)
 	}
-	fmt.Println("run list")
+	nc, err := configureNatsConnection(globals)
+	if err != nil {
+		return err
+	}
+
+	controller, err := nodecontrol.NewControlApiClient(nc, slog.New(slog.NewTextHandler(os.Stdout, nil)))
+	if err != nil {
+		return err
+	}
+
+	resp, err := controller.Ping()
+	if err != nil {
+		return err
+	}
+
+	if !l.JSON {
+		tW := newTableWriter("NATS Execution Nodes")
+		tW.AppendHeader(table.Row{"Nexus", "ID (* = Lameduck Mode)", "Name", "Version", "Agents", "Workloads", "Uptime", "OS", "Arch"})
+
+	node_loop:
+		for _, n := range resp {
+
+			// Check the filter list
+			for k, v := range l.Filter {
+				if tV, ok := n.Tags.Tags[k]; !ok || tV != v {
+					continue node_loop
+				}
+			}
+
+			lameduck, ok := n.Tags.Tags[models.TagLameDuck]
```

Comment:
Should nodes in lame duck mode even respond to pings? :thinking:

Reply:
If they are still up (draining), I think they should.  Since you can see workload count in the response, gives you an easy mechanism to watch the node drain.

Reply:
Fair enough